matrix-editor/test/sanscript.spec.js

```diff
@@ -0,0 +1,27 @@
+import {describe, it} from 'mocha';
+import {expect} from "chai";
+import {Sanscript} from "../lib/sanscript.js";
+
+describe('sanscript', () => {
+  let sanscript = null
+  beforeEach(() => {
+    sanscript = new Sanscript()
+    // TODO: use something more meaningful?
+    sanscript.romanSchemes['test'] = true
+  })
+  it('isRomanScheme', () => {
+    const tests = [
+      {
+        given: 'sardines',
+        expected: false
+      },
+      {
+        given: 'test',
+        expected: true
+      }
+    ]
+    for (const test of tests) {
+      expect(sanscript.isRomanScheme(test.given)).to.equal(test.expected)
+    }
+  })
+})
```

Comment:
An example test. Doesn't do anything super clever, but just to demonstrate that it would be possible to write simple unit tests for the functions that process the text files, prepare the data for the tree, or do simple things like merging object properties.